stdlib/2and3/pstats.pyi

```diff
@@ -0,0 +1,36 @@
+from profile import Profile
+from cProfile import Profile as cProfile
+from typing import Any, AnyStr, Dict, IO, List, Tuple, TypeVar, Union, overload
+
+_Selector = Union[AnyStr, float, int]
+_T = TypeVar('_T', bound='Stats')
+
+class Stats:
+    # As best I can tell, the first argument has to be a string or Profile
+    # instance and the remaining ones may also be Stats instances
+    # the only keyword argument accepted is "stream"
+    def __init__(self: _T, *args: Union[AnyStr, Profile, _T],
```

Comment:
Your description sounds right, but I think that you should be able to create a more precise signature still. For example:

```
def __init__(self: _T, __arg: Union[None, AnyStr, Profile] = ..., *args: Union[None, AnyStr, Profile, _T],
             stream: IO[Any] = ...): ...
```


I also used `stream: IO[Any]` instead of `**kwds`, and `None` seems like a valid argument.